spec/listformat.html

```diff
@@ -87,7 +88,7 @@ <h1>CreatePartsFromList ( _listFormat_, _list_ )</h1>
           1. Else,
             1. Let _pattern_ be _listFormat_.[[TemplateEnd]].
           1. Let _head_ be a new Record { [[Type]]: `"element"`, [[Value]]: _list_[_i_] }.
-          1. Let _tail_ be a new Record { [[Type]]: `"element"`, [[Value]]: _parts_ }.
+          1. Let _tail_ be &laquo; _parts_ &raquo;.
           1. Let _placeables_ be a new Record { [[0]]: _head_, [[1]]: _tail_ }.
```

Comment:
Remove the *tail* variable entirely, and just pass *parts* for the `[[1]]` field.

Reply:
This is not sufficient as mentioned above.